programs/mpl-core/src/instruction.rs

```diff
@@ -205,78 +206,78 @@ pub(crate) enum MplAssetInstruction {
+    /// This function creates the initial Collection, with or without internal/external plugin adapters.
     #[account(0, writable, signer, name="collection", desc = "The address of the new asset")]
     #[account(1, optional, name="update_authority", desc = "The authority of the new asset")]
     #[account(2, writable, signer, name="payer", desc = "The account paying for the storage fees")]
     #[account(3, name="system_program", desc = "The system program")]
     CreateCollectionV2(CreateCollectionV2Args),
 
-    /// Add an external plugin to an mpl-core.
+    /// Add a external plugin adapter to an mpl-core.
     #[account(0, writable, name="asset", desc = "The address of the asset")]
     #[account(1, optional, writable, name="collection", desc = "The collection to which the asset belongs")]
     #[account(2, writable, signer, name="payer", desc = "The account paying for the storage fees")]
     #[account(3, optional, signer, name="authority", desc = "The owner or delegate of the asset")]
     #[account(4, name="system_program", desc = "The system program")]
     #[account(5, optional, name="log_wrapper", desc = "The SPL Noop Program")]
-    AddExternalPluginV1(AddExternalPluginV1Args),
+    AddExternalPluginAdapterV1(AddExternalPluginAdapterV1Args),
 
-    /// Add an external plugin to an mpl-core Collection.
+    /// Add a external plugin adapter to an mpl-core Collection.
     #[account(0, writable, name="collection", desc = "The address of the asset")]
     #[account(1, writable, signer, name="payer", desc = "The account paying for the storage fees")]
     #[account(2, optional, signer, name="authority", desc = "The owner or delegate of the asset")]
     #[account(3, name="system_program", desc = "The system program")]
     #[account(4, optional, name="log_wrapper", desc = "The SPL Noop Program")]
-    AddCollectionExternalPluginV1(AddCollectionExternalPluginV1Args),
+    AddCollectionExternalPluginAdapterV1(AddCollectionExternalPluginAdapterV1Args),
 
-    /// Remove an external plugin from an mpl-core.
+    /// Remove a external plugin adapter from an mpl-core.
     #[account(0, writable, name="asset", desc = "The address of the asset")]
     #[account(1, optional, writable, name="collection", desc = "The collection to which the asset belongs")]
     #[account(2, writable, signer, name="payer", desc = "The account paying for the storage fees")]
     #[account(3, optional, signer, name="authority", desc = "The owner or delegate of the asset")]
     #[account(4, name="system_program", desc = "The system program")]
     #[account(5, optional, name="log_wrapper", desc = "The SPL Noop Program")]
-    RemoveExternalPluginV1(RemoveExternalPluginV1Args),
+    RemoveExternalPluginAdapterV1(RemoveExternalPluginAdapterV1Args),
 
-    /// Remove an external plugin from an mpl-core Collection.
+    /// Remove a external plugin adapter from an mpl-core Collection.
```

Comment:
Ultra nit "a" should have stayed "an".  Recommend find and replace "a external" with "an external"